docs/cli/how-tos/index.mdx

```diff
@@ -4,3 +4,4 @@ The following how-tos apply to the `src` command line interface to Sourcegraph:
 
 - [Creating an access token](/cli/how-tos/creating_an_access_token)
 - [Revoking an access token](/cli/how-tos/revoking_an_access_token)
+- [Managing access tokens](managing_access_tokens.mdx)
```

Comment:
I think the link should be `/cli/how-tos/managing_access_tokens` instead of `managing_access_tokens.mdx`